packages/@aws-cdk/aws-iotevents/test/integ.detector-model.ts

```diff
@@ -10,8 +10,8 @@ class TestStack extends cdk.Stack {
       attributeJsonPaths: ['payload.deviceId', 'payload.temperature'],
     });
 
-    const onlineState = new iotevents.State({
-      stateName: 'online',
+    const firstState = new iotevents.State({
```

Comment:
Can we leave the name the same here please? I mean for both the variable `onlineState`, and the `stateName` property.

Maybe call the second state `offlineState`, for symmetry?